.github/labeler.yml

```diff
@@ -1,18 +1,23 @@
 ---
 
 VisualCSharp:
-  - GetIntoTeachingApi/**/*.cs
+- changed-files:
```

Comment:
Thanks for updating this - however, the labeler is still throwing errors:

```
Run actions/labeler@main
The configuration file (path: .github/labeler.yml) was not found locally, fetching via the api
Error: Error: found unexpected type for label 'VisualCSharp' (should be array of config options)
Error: found unexpected type for label 'VisualCSharp' (should be array of config options)
```

Reply:
Yes it's a bit confusing but it turns out the "pull_request_target" trigger picks up the yml file from the master branch, so it will only work after merging.
I did test it with the "pull_request" trigger which picked up the yml file from the PR branch and it worked: https://github.com/DFE-Digital/get-into-teaching-api/actions/runs/7183796826

Reply:
Jolly good - that would explain it!